.taskcluster.yml

```diff
@@ -95,7 +96,11 @@ tasks:
           && cd  /go/src/github.com/taskcluster/taskcluster-cli
           && git fetch {{ event.head.repo.url }} {{ event.version }}
           && git checkout FETCH_HEAD
-          && make prep release upload
+          && make prep release
+      artifacts:
+        public:
+          path: /go/src/github.com/taskcluster/taskcluster-cli/taskcluster/build
+          type: directory
```

Comment:
directory is not a good idea... Use file artifacts (in some soon to be future, we'll have workers that fails tasks if artifacts are missing, if we didn't already fix that)...

Also it force a consistent naming...